README.md

```diff
@@ -10,11 +10,29 @@ This toolkit will:
 6. Log the pull request URL to merge this branch
 
 ## Installation
+From PYPI
+```shell script
+easy_install Package-Version-Manager
```

Comment:
Not possible with python -m pip install ?

Reply:
We have to use `easy_install`, there is a lot of issues about having `package_data` and using `pip`.